src/Reflection/ReflectionHelper.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
```

Comment:
Just have questions.
1. We have debates with @a-bastien about single-line methods. I am confused for now. When I should implement single-line methods? In the current case, it is the helper for the build-in php method. What is the sense in it?
2. About static methods. Is it a consequence of `Services should not use autowiring or autoconfiguration. Instead, all services should be defined explicitly` in README? More abstract: what is Your attitude to static methods in objects?

Reply:
1. Sense of this method is to have one place/method to do such a things. Once we would need to change the implementation we just need to modify one place.
2. Nope it has nothing with `autowiring or autoconfiguration`. Generally static methods should be avoided, with one exception, simple helpers like methods for string formatting, doing some math etc.